hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java

```diff
@@ -128,40 +130,31 @@ private void handleChangeClusterState(HttpPostCommand command) throws Unsupporte
                 }
                 if (!state.equals(clusterService.getClusterState())) {
                     clusterService.changeClusterState(state);
-                    res = res.replace("${STATUS}", "success");
-                    res = res.replace("${STATE}", state.toString().toLowerCase());
+                    res = response(ResponseType.SUCCESS, "state", state.toString().toLowerCase());
                 } else {
-                    res = res.replace("${STATUS}", "fail");
-                    res = res.replace("${STATE}", state.toString().toLowerCase());
+                    res = response(ResponseType.FAIL, "state", state.toString().toLowerCase());
                 }
             }
         } catch (Throwable throwable) {
-            res = res.replace("${STATUS}", "fail");
-            res = res.replace("${STATE}", "null");
+            logger.severe("Error occurred while changing cluster state", throwable);
```

Comment:
Do we want to have this on severe level? Normally I use severe for some kind of internal system error. In this case exceptions could be thrown because the user did something 'stupid'.